pkg/ddl/ddl.go

```diff
@@ -416,20 +449,51 @@ func (sv *schemaVersionManager) setSchemaVersion(job *model.Job, store kv.Storag
 }
 
 // lockSchemaVersion gets the lock to prevent the schema version from being updated.
-func (sv *schemaVersionManager) lockSchemaVersion(jobID int64) {
+func (sv *schemaVersionManager) lockSchemaVersion(jobID int64) error {
 	ownerID := sv.lockOwner.Load()
 	// There may exist one job update schema version many times in multiple-schema-change, so we do not lock here again
 	// if they are the same job.
 	if ownerID != jobID {
 		sv.schemaVersionMu.Lock()
 		sv.lockOwner.Store(jobID)
+		if sv.etcdClient != nil && variable.DDLVersion.Load() == model.TiDBDDLV2 {
+			se, err := concurrency.NewSession(sv.etcdClient)
+			if err != nil {
+				return errors.Trace(err)
+			}
+			mu := concurrency.NewMutex(se, ddlSchemaVersionKeyLock)
```

Comment:
Could we add a simple comment for this lock？

Reply:
[af4f63d](https://github.com/pingcap/tidb/pull/49687/commits/af4f63d5efb936bb1911c60b9819ac568fa05dcb)